controllers/util/mcv_util.go

```diff
@@ -296,30 +304,41 @@ Requires:
 
 Returns: ManagedClusterView, error
 */
-func (m ManagedClusterViewGetterImpl) getOrCreateManagedClusterView(
+func (m ManagedClusterViewGetterImpl) getOrApplyManagedClusterView(
 	meta metav1.ObjectMeta, viewscope viewv1beta1.ViewScope, logger logr.Logger,
 ) (*viewv1beta1.ManagedClusterView, error) {
+	key := types.NamespacedName{Name: meta.Name, Namespace: meta.Namespace}
 	mcv := &viewv1beta1.ManagedClusterView{
 		ObjectMeta: meta,
 		Spec: viewv1beta1.ViewSpec{
 			Scope: viewscope,
 		},
 	}
 
-	err := m.Get(context.TODO(), types.NamespacedName{Name: meta.Name, Namespace: meta.Namespace}, mcv)
+	err := m.Get(context.TODO(), key, mcv)
 	if err != nil {
 		if errors.IsNotFound(err) {
-			logger.Info(fmt.Sprintf("Creating ManagedClusterView %v", mcv))
+			logger.Info(fmt.Sprintf("Creating ManagedClusterView %s with scope %+v",
+				key, viewscope))
+
 			err = m.Create(context.TODO(), mcv)
 		}
 
 		if err != nil {
-			return nil, errorswrapper.Wrap(err, "failed to getOrCreateManagedClusterView")
+			return nil, errorswrapper.Wrap(err, "failed to create ManagedClusterView")
 		}
 	}
 
 	if mcv.Spec.Scope != viewscope {
-		logger.Info("WARNING: existing ManagedClusterView has different ViewScope than desired one")
+		// Expected once when uprading ramen if scope format or details have
+		// changed.
```

Comment:
Text is hard to read with long lines. For code we sometimes need long lines to deal with long variables names and functions and too much indentation. For comments there is never
need to use long lines.

Reply:
@nirs I would rather not debate ideal length of lines in code, just so you know it is ~120 chars in all files including comments. So it would be useful to stick to the same, rather than introducing style differences.

Reply:
I'm also not interested in debating this, but when I write new code or update exiting code I adapt it to be easy to read, since I cannot work with longer lines.

Reply:
Fortunately I have never seen a project cater to individual styles and preferences, and I do not see why we should start here. This goes against the project accepted style and further is forcing your individual style now and in the future as per your comment.

Instead of leaving a counter example in the code here I request you to address this and implore that we follow the project style in spirit as much as possible. 

NOTE: FWIW, this shows up in my radar because there is a single word in the next comment line as otherwise I would have missed your entire intention of sticking to 80 chars in the future as well.